source/lex.tex

```diff
@@ -334,8 +334,8 @@
 circumstances during translation phase 4, white space (or the absence
 thereof) serves as more than preprocessing token separation. White space
 can appear within a preprocessing token only as part of a header name or
-between the quotation characters in a character literal or string
-literal.
+between the quotation characters in a \grammarterm{character-literal} or
+\grammarterm{string-literal}.
```

Comment:
This is also unclear in the presence of UDLs.

Reply:
Reverted.